src/main/resources/view/DarkTheme.css

```
@@ -147,10 +147,11 @@
 }
 
 .result-display {
-    -fx-background-color: transparent;
+    -fx-background-color: #222222;
     -fx-font-family: "Segoe UI Light";
     -fx-font-size: 13pt;
     -fx-text-fill: white;
+    -fx-padding: 18;
 }
```

As suggested by @ichigh0st, I think we should wrap the text in the `resultDisplay` to prevent horizontal scrolling.
```suggestion
    -fx-wrap-text: true;
}
```